mass-deployer/pkg/mass-deployer/node_filter.go

```diff
@@ -11,7 +11,7 @@ func filterNodes(tfPluginClient deployer.TFPluginClient, group NodesGroup, ctx c
 	filter := types.NodeFilter{}
 
 	statusUp := "up"
-	freeMRU := convertMBToBytes(group.FreeMRU)
+	freeMRU := convertMBToBytes(uint64(group.FreeMRU * 1024))
```

Comment:
just use convert to gb

Reply:
group.FreeMRU is float fa it wouldn't work as expected